spaces/S000168/properties/P000123.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000168
+property: P000123
+value: true
+---
+
+Each point in {S168} has a small neighborhood which contains no antipodal point pair, which is invariant under quotient map, and the result follows by {S169|P123}.
```

Comment:
Perhaps it will be clearer to say every point in the sphere (instead of in the projective plane) has such a small neighborhood and to say the quotient map restricts to a homeomorphism from the said neighborhood to an open set in the projective plane?